CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Welcome! 🎉
+
+Welcome to the Electricity Maps open source contribution repository. </br>
+Any and all contributions however big or small are welcome.
+
+# Contribution Guidelines
```

Comment:
Maybe we should also use this file to share a bit more about:
- When to create an issue vs discussion
- How we like PRs (small, fixing one thing, documented/explained in description, etc.)

I personally think https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md has some nice phrasings for inspiration :)